diff --git a/setup.py b/setup.py
index 946ee91a41..703048abbf 100644
--- a/setup.py
+++ b/setup.py
@@ -1528,6 +1528,7 @@ def detect_decimal(self):
                 ['_decimal/_decimal.c'],
                 # Uncomment for extra functionality:
                 # define_macros=[('EXTRA_FUNCTIONALITY', 1)]
+                extra_compile_args=['-Wno-error=implicit-function-declaration']
             )
         )
 
