diff --git a/Modules/_cursesmodule.c b/Modules/_cursesmodule.c
index 45f3d81ec0..ba3af03478 100644
--- a/Modules/_cursesmodule.c
+++ b/Modules/_cursesmodule.c
@@ -675,7 +675,7 @@ PyCursesWindow_New(WINDOW *win, const char *encoding)
 
     if (encoding == NULL) {
 #if defined(MS_WINDOWS)
-        char *buffer[100];
+        char buffer[100];
         UINT cp;
         cp = GetConsoleOutputCP();
         if (cp != 0) {
